pkg/agent/datamodel/types.go

```diff
@@ -622,6 +622,22 @@ type KubernetesAddon struct {
 	Data       string                    `json:"data,omitempty"`
 }
 
+// EbpfDataplane controls the eBPF networking dataplane.
+type EbpfDataplane int32
+
+const (
+	// none means don't install an eBPF dataplane.
+	EbpfDataplane_none EbpfDataplane = 0
```

Comment:
nit: I'd use golang's iota feature here instead: https://go.dev/wiki/Iota

Reply:
eh, I guess what you have is fine if you're trying to mirror RP

Reply:
Leaving this untouched.